src/ioctls/vcpu.rs

```diff
@@ -8,13 +8,14 @@
 use kvm_bindings::*;
 use libc::EINVAL;
 use std::fs::File;
-use std::io;
 use std::os::unix::io::{AsRawFd, RawFd};
 
-use ioctls::{KvmRunWrapper, Result};
+use ioctls::KvmRunWrapper;
 #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
 use kvm_bindings::{CpuId, Msrs};
 use kvm_ioctls::*;
+use vmm_sys_util::errno::Error as Errno;
+use vmm_sys_util::errno::Result as ErrnoResult;
```

Comment:
I noticed that in the other commits you use errno::Error. I think it's better to be consistent here as well and import `vmm_sys_util::errno` instead.

Reply:
Oh, this is leftover code, good catch!